pytorch_lightning/loggers/wandb.py

```diff
@@ -468,6 +496,47 @@ def after_save_checkpoint(self, checkpoint_callback: "ReferenceType[ModelCheckpo
         elif self._log_model is True:
             self._checkpoint_callback = checkpoint_callback
 
+    @classmethod
```

Comment:
LGTM but fyi cls is not used here. Does it have to be a class method? Could also be static